src/2d/shallow/topo_module.f90

```diff
@@ -600,6 +602,46 @@ subroutine read_topo_file(mx,my,topo_type,fname,topo)
                     topo(j * mx + 1:(j + 1) * mx) = nc_buffer(:, my - j)
                 end do
                 deallocate(nc_buffer)
+
```

Comment:
I think you may have added this on accident.  Not opposed to having this added but wanted to just check in to see what the purpose is.

Reply:
yes like you alluded to in the PR comment, this is for the netcdf filling missing value functionality.